site/docs/Learn-Schema.md

```diff
@@ -85,7 +90,7 @@ That means that the GraphQL service needs to have a `Query` type with `hero` and
 ```graphql
 type Query {
   hero(episode: Episode): Character
-  droid(id: ID!): Droid
+  droid(id: String!): Droid
```

Comment:
I don't see why that is a best practice - the only benefit of ID is that you can pass it an int and it will automatically cast to a string, right? normally you would not want to do that in your schema AFAICT. if it's a string in your database it seems safer to have it be a string in graphql.